docs/CHANGELOG.rst

```diff
@@ -24,7 +24,7 @@ Changed
 
 Fixed
 -----
-
+- Ensure the sorting of input VCF file in ``variant-annotation`` process
```

Comment:
Changelog entry is not needed, as it describes a change to yet unreleased feature.